assignment2/style.css

```diff
@@ -0,0 +1,103 @@
+body {
+  font-family: "Raleway", sans-serif;
+  margin: 0;
+  padding: 0;
+  color: rgb(153, 149, 149);
+}
+h1 {
+  font-weight: 400;
+  padding-top: 30px;
+  margin-left: 125px;
+  color: rgb(51, 50, 50);
+}
+h2 {
+  color: hsl(0deg 0% 20%);
+}
+h3 {
+  font-weight: 700;
+  color: hsl(0deg 0% 20%);
+}
+p{
+  font-size:18px;
+}
+.box {
+  display: flex;
+}
+.inner-box {
+  align-items: center;
+  display: flex;
+  padding: 10px;
+  text-align: center;
+  width: 52%;
+  margin-left: 120px;
+  margin-top: -10px;
+}
+.link{
+  text-decoration: none;
+  color: rgb(38, 147, 219);
+}
+.first {
+  height: 300px;
+  width: 332px;
+  padding-top: 15px;
+  margin-bottom: -50px;
+}
+.shift {
+  margin-left: 30px;
+}
+.image{
+  height: 50px;
+  width: 50px;
+}
+.form-box {
+  padding: 5px;
+  margin-top: -40px;
+}
+.select {
+  padding-left: 46px;
+  text-align: left;
+  color: hsl(0deg 0% 20%);
+  font-weight: 400;
+}
+.choose {
+  margin-left: -50px;
+}
+.choose button {
+  background-color: white;
+  border: 1px solid rgba(53, 51, 51, 0.123);
+  color: rgba(34, 31, 31, 0.445);
+  padding: 10px;
+  width: 82px;
+  height: 45px;
+  font-size: 15px;
+}
+.details {
+  box-shadow: -2px -2px 40px 0 rgb(154, 150, 150);
+  padding: 30px 1px;
+  text-align: center;
+  width: 400px;
+}
+.user-data {
+  border: 1px solid rgba(53, 51, 51, 0.123);
+  margin: 10px 0px;
+  margin-left: -30px;
+  padding: 20px;
+  width: 60%;
+  font-size: 15px;
+}
+.submit{
+  background-color: hsl(183, 52.63%, 52.51%);
+  border: none;
+  color: white;
+  font-size: 18px;
+  font-weight: 600;
+  margin-top: 15px;
+  padding: 15px;
+  border-radius: 5px;
+  width: 70%;
+  margin-left:-30px;
+}
+.contact {
+  font-family: sans-serif;
+  color: rgba(34, 31, 31, 0.445);
+}
```

Comment:
get this red symbol removed by adding newline at the end of each file you work on.